stories/constants/boxShadows.stories.tsx

```diff
@@ -2,60 +2,41 @@ import React from 'react';
 import { css } from '@emotion/core';
 import styled from '@emotion/styled';
 import { SPACER } from 'src/constants';
-import * as BOX_SHADOWS from 'src/constants/boxShadows';
-import { Container, Typography } from 'src/shared-components';
+import { Container } from 'src/shared-components';
+import { useTheme } from 'emotion-theming';
 
 const MainContainer = styled.div`
   padding: ${SPACER.xlarge};
-`;
-
-const BoxesContainer = styled.div`
-  margin: ${SPACER.xlarge} 0;
   display: flex;
   flex-flow: row wrap;
   justify-content: space-around;
   align-items: center;
 `;
 
-/* eslint-disable-next-line @typescript-eslint/no-unused-vars */
-const { default: defaultExport, ...VALID_BOX_SHADOWS } = BOX_SHADOWS;
-
 export const BoxShadows = () => {
-  const boxShadowKeys = Object.keys(VALID_BOX_SHADOWS) as Array<
-    keyof typeof VALID_BOX_SHADOWS
+  const theme = useTheme();
+  const boxShadowsKeys = Object.keys(theme.BOX_SHADOWS) as Array<
+    keyof typeof theme['BOX_SHADOWS']
   >;
 
   return (
     <MainContainer>
-      {boxShadowKeys.map((category) => {
-        const categoryBoxShadows = VALID_BOX_SHADOWS[category];
+      {boxShadowsKeys.map((boxShadowKey) => {
+        const boxShadowValue = theme.BOX_SHADOWS[boxShadowKey];
+        const boxShadowBoxStyles = css`
+          width: 350px;
+          margin: ${SPACER.large};
+          box-shadow: ${boxShadowValue};
+        `;
 
         return (
-          <React.Fragment key={category}>
-            <Typography.Title>{category}:</Typography.Title>
-            <BoxesContainer>
-              {Object.keys(categoryBoxShadows).map((shadow) => {
-                const boxShadowValue = categoryBoxShadows[shadow] as string;
-
-                const styles = css`
-                  width: 350px;
-                  margin: ${SPACER.large};
-                  box-shadow: ${boxShadowValue};
-                `;
-
-                return (
-                  <Container key={shadow} css={styles}>
-                    <Container.Section>
-                      <strong>Key:</strong> {shadow}
-                      <br />
-                      <br />
-                      <strong>Value:</strong> {boxShadowValue}
-                    </Container.Section>
-                  </Container>
-                );
-              })}
-            </BoxesContainer>
-          </React.Fragment>
+          <Container key={boxShadowKey} css={boxShadowBoxStyles}>
+            <Container.Section>
+              <strong>Key:</strong> {boxShadowKey}
+              <br />
```

Comment:
[Nit] I think this looks better with two:

<img width="386" alt="Screen Shot 2020-12-07 at 3 09 41 PM" src="https://user-images.githubusercontent.com/13544620/101405792-40517300-389e-11eb-80cc-598ffc31185b.png">
<img width="382" alt="Screen Shot 2020-12-07 at 3 09 37 PM" src="https://user-images.githubusercontent.com/13544620/101405795-40ea0980-389e-11eb-86fc-338de808b65c.png">

Reply:
ok added a br